src/player/PlayerComponent.cpp

```diff
@@ -282,6 +297,13 @@ void PlayerComponent::queueMedia(const QString& url, const QVariantMap& options,
   extraArgs.insert("ad", "");
   extraArgs.insert("vd", "");
 
+  extraArgs.insert("tls-verify", "yes");
+  extraArgs.insert("stream-lavf-o", "verifyhost=" + host);
```

Comment:
`tls-verify` and `tls-ca-file` can be; `verifyhost`, probably not, since it could differ between queue items?